lua/neoclip/settings.lua

```diff
@@ -78,11 +78,19 @@ local function check_deprecated_entries()
 end
 
 local function setup(opts, subsettings)
+    --- `is_setable` returns `true` for dict-like and empty tables.
+    --- Thus, allowing to filter out array-like tables.
+    local function is_setable (t)
+        return type(t) == 'table'
+            and not (#t > 0 and next(t, #t) == nil)
```

Comment:
```suggestion
            and not vim.tbl_islist(t)
```
this would be the same right? and a bit cleaner

Reply:
Thx! It is definitely cleaner